acceptancetests/jujupy/client.py

```diff
@@ -2429,9 +2429,26 @@ def _wait_for_status(self, reporter, translate, exc_type=StatusNotMet,
             reporter.finish()
         return status
 
+    def wait_for_application(self, application, timeout=60):
+        """Wait till config returns without error. This is used as a
+        proxy for the application itself being ready to accept
+        operations.
+
+        :param application: Application to wait for
+        """
+        for ignored in until_timeout(timeout):
+            try:
+                return self.get_config(application)
+            except subprocess.CalledProcessError:
+                time.sleep(10)
+                pass
+        raise Exception(
+            'Timed out waiting for %s' % (application))
```

Comment:
Please use str.format() it's nicer to use and matches what's used elsewhere throughout the project.